yodeploy/test_integration/samples/nginx-app/deploy/hooks.py

```diff
@@ -0,0 +1,20 @@
+import getpass
+
+import mock
+
+from os.path import join
+
+from yodeploy.hooks.nginx import NginxHostedApp
+from yodeploy.test_integration.helpers import tests_dir
+
+
+class Hooks(NginxHostedApp):
+    # these overrides are only needed for testing
+    nginx = mock.Mock()
+    logs_path = join(tests_dir, 'filesys/var/log/nginx')
+    server_blocks_path = join(tests_dir, 'filesys/etc/nginx/sites-enabled')
```

Comment:
Why it is called this way? Not "enabled_sites"?

Reply:
It is nginx terminology. https://www.nginx.com/resources/wiki/start/topics/examples/server_blocks/
So I've decided to name it server block instead of vhost (which is Apache terminology).